server/pairing/sync_device_test.go

```diff
@@ -1167,3 +1167,99 @@ func (s *SyncDeviceSuite) TestTransferringKeystoreFilesAfterStopUisngKeycard() {
 		require.True(s.T(), containsKeystoreFile(clientKeystorePath, acc.Address.String()[2:]))
 	}
 }
+
+func (s *SyncDeviceSuite) TestPreventLoggedInAccountLocalPairingClientAsReceiver() {
+	clientTmpDir := filepath.Join(s.clientAsSenderTmpdir, "client")
+	clientBackend := s.prepareBackendWithAccount("", clientTmpDir)
+	serverTmpDir := filepath.Join(s.clientAsSenderTmpdir, "server")
+	serverBackend := s.prepareBackendWithAccount("", serverTmpDir)
+	defer func() {
+		s.NoError(serverBackend.Logout())
+		s.NoError(clientBackend.Logout())
+	}()
+
+	serverActiveAccount, err := serverBackend.GetActiveAccount()
+	s.NoError(err)
+	serverKeystorePath := filepath.Join(serverTmpDir, keystoreDir, serverActiveAccount.KeyUID)
+	var config = &SenderServerConfig{
+		SenderConfig: &SenderConfig{
+			KeystorePath: serverKeystorePath,
+			DeviceType:   "desktop",
+			KeyUID:       serverActiveAccount.KeyUID,
+			Password:     s.password,
+		},
+		ServerConfig: new(ServerConfig),
+	}
+	configBytes, err := json.Marshal(config)
+	s.NoError(err)
+	cs, err := StartUpSenderServer(serverBackend, string(configBytes))
+	s.NoError(err)
+
+	expectedKDFIterations := 2048
```

Comment:
nit: would be good to have expected KDF values for mobile and desktop in some constant :)